src/lib/ai/datasets/patient-psi-integration.ts

```
@@ -0,0 +1,998 @@
+  FormativeExperience,
+  TherapyHistory,
+  ConversationalStyle,
+  DemographicInfo,
+  DiagnosisInfo,
+  TherapeuticProgress,
+} from '../types/CognitiveModel'
+import type { PatientPsiIndexer } from './patient-psi-indexer'
+import {
+  PatientPsiParser,
+  type PatientPsiCognitiveModel as ParsedPatientModel,
+} from './patient-psi-parser'
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)